package.json

```diff
@@ -103,6 +103,21 @@
           "default": "",
           "description": "%ext.config.gist%"
         },
+        "sync.repoEnabled": {
```

Comment:
Repo config should be available in Settings sync global settings.
 https://github.com/shanalikhan/code-settings-sync#global-settings

We dont require the Repo configs in the `settings.json`.
1 Use case : If user select `gist` form the config, settings.json will be stored in the gist if user select `repo` then it should be pushed to repo

So it would be great to move repo configs in global config file with s switch
`sync-mode` : `gist` or `repo`
You can create new class for the repo configs in put it as a member of custom settings
https://github.com/shanalikhan/code-settings-sync/blob/master/src/setting.ts#L31

Reply:
so `sync-mode` should be placed in global settings but what about `repoPush` and `repoUrl`? They stay here?

Reply:
They should also move to global settings.

Reply:
if gist setting (url) stays here so why repo url not?

Reply:
Yes repo url can stay here., :)